packages/preview-middleware/templates/flp/editor.html

```diff
@@ -5,9 +5,8 @@
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
         <meta name="description" content="Web page for rendering SAP Fiori Tools Control Property Editor" />
-        <link rel="shortcut icon" type="image/x-icon" href="./editor/favicon.ico" />
```

Comment:
We still need to be able to add a favicon, perhaps as another optional start parameter with URI for the icon.

Reply:
In which case do we still need it?

Reply:
Never mind my comment, I was not aware that there was a decision to remove the icon.